### PR TITLE
Update to use airbnb eslint config

### PR DESCRIPTION
**Pull request by goatslacker**
ref #473

### CODE REVIEW

**Comment by goatslacker** [comment]
cc @taion @mull

File: .eslintrc [hunk 1]
```diff
@@ -13,172 +14,8 @@
...
-      "space-return-throw-case": 2,
-      "spaced-line-comment": 2,
-
-    /* custom rules */
-    "no-console": 0,
-    "space-before-function-paren": [2, {
-      "named": "never",
-      "anonymous": "always"
-    }],
     "semi": [2, "never"],
```
Comment (taion) [inline .eslintrc]: :smile:

File: src/alt/store/StoreMixin.js [hunk 2]
```diff
@@ -46,7 +46,9 @@ const StoreMixin = {
         const value = spec.local && spec.local(state, ...args)
         const shouldFetch = spec.shouldFetch
           ? spec.shouldFetch(state, ...args)
+          /*eslint-disable*/
```
Comment (taion) [inline src/alt/store/StoreMixin.js]: Didn't you already disable `no-eq-null`?

**Comment by taion** [comment]
You should probably add either `eslint:recommended` to the `extends`, or manually add `no-undef`. Or get someone to release `eslint-config-airbnb@0.0.9`.

File: src/alt/store/StoreMixin.js (hunk 2 above)
Comment (goatslacker) [inline src/alt/store/StoreMixin.js]: yeah but it's complaining that I'm not using triple equals. I think it's a bug in eslint.

Comment (taion) [inline src/alt/store/StoreMixin.js]: Bizarre.

**Comment by goatslacker** [comment]
I think it's easier to release eslint config airbnb :)

**Pull request merged**